### ISSUE TITLE
Ben's plan for the summer

### ISSUE THREAD

**Issue by llotneb** [clarityinc/clarityeco#47]
- [x] 7/30 - 7/31: Finish documentation, clean-up.
- [x] 8/3 - 8/7: Any additional wrap-up work I'm forgetting.

**Comment by llotneb** [comment]
Below is a summary of my work this summer. The current status of the OpenEphyra project is described under "OE Enhancements", and my wrap-up plans are listed above.

- I wrote a wrapper around the OpenEphyra question-answering system that allows OpenEphyra to communicate with other Sirius components through a unified Apache thrift interface.
- Based on Tim's design, I implemented a simple command center to register Sirius services and forward requests between them.
- I added regular expression parsing for Kaldi audio transcripts. *(Someone will need to separate this function from the command center implementation for the next release).*
- I help port our code to a new clarityeco repo, and helped other teammates get up-to-speed on the new commit practices, my code, etc.
- I demoed sirius with other members of the team.
